contributors/devel/pull-requests.md

```diff
@@ -1,111 +1,326 @@
+1. Automatic tests run. See the current list of tests on the [MERGE REQUIREMENTS tab, at this link](https://submit-queue.k8s.io/#/info)
+1. If tests fail, resolve issues by pushing edits to your PR branch
+1. If the failure is a flake, a member can comment `@k8s-bot [e2e|unit] test this issue: #<flake issue>`
 
-1. All pull requests are initiated with a `release-note-label-needed` label.
-1. For a PR to be ready to merge, the `release-note-label-needed` label must be removed and one of the other `release-note-*` labels must be added.
-1. `release-note-none` is a valid option if the PR does not need to be mentioned
- at release time.
-1. `release-note` labeled PRs generate a release note using the PR title by
-   default OR the release-note block in the PR template if filled in.
-  * See the [PR template](https://github.com/kubernetes/kubernetes/blob/master/.github/PULL_REQUEST_TEMPLATE.md) for more
-    details.
-  * PR titles and body comments are mutable and can be modified at any time
-    prior to the release to reflect a release note friendly message.
+Once the tests pass, all failures are commented as flakes, or the reviewer adds the labels `lgtm` and `approved`, the PR enters the final merge queue. The merge queue is needed to make sure no incompatible changes have been introduced by other PRs since the tests were last run on your PR.
 
-The only exception to these rules is when a PR is not a cherry-pick and is
-targeted directly to the non-master branch.  In this case, a `release-note-*`
-label is required for that non-master PR.
+Either the [on call contributor](on-call-rotations.md) will manage the merge queue manually, or the [GitHub "munger"](https://github.com/kubernetes/test-infra/tree/master/mungegithub) submit-queue plugin will manage the merge queue automatically.
 
-### Reviewing pre-release notes
+1. The PR enters the merge queue ([http://submit-queue.k8s.io](http://submit-queue.k8s.io))
+1. The merge queue triggers a test re-run with the comment `@k8s-bot test this`
+    1. Author has signed the CLA (`cla: yes` label added to PR)
+    1. No changes made since last `lgtm` label applied
+1. If tests fail, resolve issues by pushing edits to your PR branch
+1. If the failure is a flake, a member can comment `@k8s-bot [e2e|unit] test this issue: #<flake issue>`
+1. If tests pass, the merge queue automatically merges the PR
 
-At any time, you can see what the release notes will look like on any branch.
-(NOTE: This only works on Linux for now)
+That's the last step. Your PR is now merged.
 
-```
-$ git pull https://github.com/kubernetes/release
-$ RELNOTES=$PWD/release/relnotes
-$ cd /to/your/kubernetes/repo
-$ $RELNOTES -man # for details on how to use the tool
-# Show release notes from the last release on a branch to HEAD
-$ $RELNOTES --branch=master
-```
+## Comment Commands Reference
 
-## Visual overview
+Refer to the comment commands [in the commands doc](https://github.com/kubernetes/test-infra/blob/master/commands.md).
```

Comment:
Updated to:

"[The commands doc](https://github.com/kubernetes/test-infra/blob/master/commands.md) contains a reference for all comment commands."

It is for reference.